src/rust/storage/seg/src/datapool/file.rs

```diff
@@ -20,30 +20,57 @@ pub struct File {
 }
 
 impl File {
-    /// Create a new `File` datapool at the given path and with the specified
-    /// size (in bytes). Returns an error if the file already exists, could not
-    /// be created, couldn't be extended to the requested size, or couldn't be
+    /// If there is a file at the given path, open the `File`.
```

Comment:
I don't think this comment matches the behavior anymore? Update the docs and improve the formatting here, it should wrap at 80 characters wide. I think some of your lines are too short, and others too long